config/settings/tailer.q

```diff
@@ -0,0 +1,63 @@
+                                                                                            // 2. partbyattr       - the data is partitioned by [ partitiontype ] and the column(s)assigned the parted attributed in sort.csv
+                                                                                            //                       at EOD the data will be merged from each partiton before being moved to hdb
+												
+mergenumrows:100000                                                                         // default number of rows for merge process
+mergenumtab:`quote`trade!10000 50000                                                        // specify number of rows per table
+
+tpconnsleepintv:10                                                                          // number of seconds between attempts to connect to the tp
+upd:insert                                                                                  // value of the upd function
+replay:1b                                                                                   // replay the tickerplant log file
+schema:1b                                                                                   // retrieve schema from tickerplant
+settimer:0D00:00:10                                                                         // timer to check if data needs written to disk
+partitiontype:`date                                                                         // set type of partition (defaults to `date, can be `date, `month or `year)
+gmttime:1b                                                                                  // define whether the process is on gmttime or not
+getpartition:{@[value;`.wdb.currentpartition;(`date^partitiontype)$(.z.D,.z.d)gmttime]}     // function to determine the partition value
+reloadorder:`hdb`rdb                                                                        // order to reload hdbs and rdbs
+hdbdir:`:hdb                                                                                // move wdb database to different location
+sortcsv:hsym first .proc.getconfigfile"sort.csv"                                            // location of csv file
+permitreload:1b                                                                             // enable reload of hdbs/rdbs
+compression:()                                                                              // specify the compress level, empty list if no required
+gc:1b                                                                                       // garbage collect at appropriate points (after each table save and after sorting data)
+eodwaittime:0D00:00:10.000                                                                  // time to wait for async calls to complete at eod
+tpcheckcycles:0W                                                                            // number of attempts to connect to tp before process is killed
+
+\d .ds
+datastripe:0b
```

Comment:
Should datastriping be disabled by default? Mentioned above, but my understanding of tailer is that it only makes sense when we're using datastriping (otherwise we'd just use the regular wdb), so not sure it makes sense that datastriping is turned off by default

Reply:
At the moment I have not included wdbs in the start all for torq5, as such they will not be running by default, this is why I thought it was okay for tailers to accept unstriped data.

would it be better to have the wdbs starting as well, then stop either the wdbs, if datastriping is off, or the tailers, if it is on?

this would also remove the need for the previous ifs.

Reply:
Perhaps I'm confused but I think tailers should always have datastriping on, and then we control whether we're using tailers (& therefore datastriping) or wdb(s) (& therefore no datastriping) by which we processes we start (i.e. by changing the start all stuff in process.csv)

To me, it seems confusing if we have the wdb as a separate proctype, but the tailer can also be used without datastriping - in which case it does the same job as a wdb(?), so why would we need to have both existing?

Reply:
Okay, that sounds good